webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+                    "Parameters included here are not guaranteed to be included in the output model."
+                ),
+            },
+            {
+                "id": self.uuid("interaction"),
+                "content": (
+                    "Select the depth of the interaction level. 'Off' allows only for the parameters "
+                    "in their original state. '2 levels' allow for the product of two original "
+                    "parameters. '3 levels' allow for the product of three original parameters. "
+                    "This feature allows you to investigate possible feedback effects."
+                ),
+            },
+            {
+                "id": self.uuid("max-params"),
+                "content": (
+                    "Choose the maximum number of parameters to include in your model. If "
+                    "interaction is active, the number of included parameters is the selected value "
+                    "here plus the interaction level. This is to make sure the interaction terms "
+                    "have an intuitive interpretation."
+                ),
+            },
+            {
+                "id": self.uuid("force-in"),
+                "content": (
+                    "Select parameters to force into the model."
+                ),
+            },
+            {
+                "id": self.uuid("submit-button"),
+                "content": (
+                    "Press this button to update the table and the plots based on the settings above."
+                ),
+            },
+        ]
+        return steps
+
+    @property
+    def responses(self):
+        """ Returns valid responses. Filters out non numerical and filterable columns. """
+        responses = list(
+            self.responsedf.drop(["ENSEMBLE", "REAL"], axis=1)
+            .apply(pd.to_numeric, errors="coerce")
+            .dropna(how="all", axis="columns")
+            .columns
+        )
+        return [p for p in responses if p not in self.response_filters.keys()]
+
+    @property
+    def parameters(self):
+        """ Returns numerical input parameters """
+        parameters = list(
+            self.parameterdf.drop(["ENSEMBLE", "REAL"], axis=1)
+            .apply(pd.to_numeric, errors="coerce")
+            .dropna(how="all", axis="columns")
+            .columns
+        )
+        return parameters
+
+    @property
+    def ensembles(self):
+        """ Returns list of ensembles """
+        return list(self.parameterdf["ENSEMBLE"].unique())
+
+    @property
+    def colors(self):
+        """Dictionary of colors that are frequently used"""
+        fig = go.Figure().to_dict()
+        fig["layout"] = self.theme.create_themed_layout(fig["layout"])
+        return {
+            "default color": fig["layout"]["colorway"][0],
+            "gray": "#606060",
+            "dark gray": "#303030",
+            "default text": fig["layout"]["template"]["layout"]["font"]["color"]
```

Comment:
Should work if you follow the suggestion on skipping the text font color later
```suggestion
        return {
            "default color": self.theme.plotly_theme["layout"]["colorway"][0],
            "gray": "#606060",
            "dark gray": "#303030",
```